packages/core/admin/server/routes/serve-admin-panel.js

```diff
@@ -46,6 +46,13 @@ const serveStatic = (filesDir, koaStaticOptions = {}) => {
   return async (ctx, next) => {
     const prev = ctx.path;
     const newPath = path.basename(ctx.path);
+    const ext = path.extname(ctx.path);
+
+    // publicly cache static files to avoid unnecessary network & disk access
+    if (ext === '.css' || ext === '.js') {
```

Comment:
- Do you think we could use the setHeaders method of koaStatic instead?
- Should we also cache fonts and svgs as they are hashed the same way?

Reply:
Hi @alexandrebodin !
- `setHeaders` from koaStatic is now used!
- Good catch about fonts and svgs. After looking more into what is built on `dist` folder, I've found that all files can be cached except `.html` files, can you confirm?


I also ensured that this does not affect `strapi develop --watch-admin`